pkg/logs/launchers/integration/launcher.go

```diff
@@ -154,10 +154,10 @@ func (s *Launcher) createFile(source *sources.LogSource) (string, error) {
 
 // integrationLogFilePath returns a directory and file to use for an integration log file
 func (s *Launcher) integrationLogFilePath(source sources.LogSource) (string, string) {
-	fileName := source.Name + ".log"
+	fileName := source.Config.Name + ".log"
 	directoryComponents := []string{s.runPath, "integrations", source.Config.Service}
 	directory := strings.Join(directoryComponents, "/")
-	filepath := strings.Join([]string{directory, fileName}, "")
+	filepath := strings.Join([]string{directory, fileName}, "/")
```

Comment:
Can we add a UT to cover this function to prevent such an error in the future?

Reply:
Added!